src/resources/nfts/index.ts

```diff
@@ -56,9 +60,65 @@ function fetchPolygonAllowlist() {
   );
 }
 
-export function useNFTs(): NFT[] {
-  // normal react query where we get new NFT formatted data
-  return [];
+export function useNFTs({ address }: { address: string }) {
+  const { wallets } = useSelector((state: AppState) => state.wallets);
+
+  const walletAddresses = useMemo(
+    () =>
+      wallets
+        ? Object.values(wallets).flatMap(wallet =>
+            wallet.addresses.map(account => account.address)
+          )
+        : [],
+    [wallets]
+  );
+  const isImportedWallet = walletAddresses.includes(address);
```

Comment:
Did we talk about this? Looking at this now, I think we should keep these resources as purely data as much as possible. We can pass in `isImportedWallet` instead. Would that work?

Reply:
agree